Gemfile.lock

```diff
@@ -184,25 +184,25 @@ GEM
     turbolinks (5.2.1)
       turbolinks-source (~> 5.2)
     turbolinks-source (5.2.0)
-    tzinfo (1.2.6)
+    tzinfo (1.2.7)
       thread_safe (~> 0.1)
     uglifier (4.2.0)
       execjs (>= 0.3.0, < 3)
-    web-console (4.0.1)
+    web-console (4.0.4)
       actionview (>= 6.0.0)
       activemodel (>= 6.0.0)
       bindex (>= 0.4.0)
       railties (>= 6.0.0)
-    webdrivers (4.2.0)
+    webdrivers (4.4.1)
       nokogiri (~> 1.6)
       rubyzip (>= 1.3.0)
       selenium-webdriver (>= 3.0, < 4.0)
-    websocket-driver (0.7.1)
+    websocket-driver (0.7.3)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
     xpath (3.2.0)
       nokogiri (~> 1.8)
-    zeitwerk (2.3.0)
+    zeitwerk (2.4.0)
```

Comment:
### [zeitwerk](https://github.com/fxn/zeitwerk)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.3.0 → 2.4.0

[_[change-log](https://github.com/fxn/zeitwerk/blob/master/CHANGELOG.md), [source-code](https://github.com/fxn/zeitwerk)_]





<details>
<summary>Commits</summary>

A change of **21** commits. See the full changes on [the compare page](https://github.com/fxn/zeitwerk/compare/v2.3.0...v2.4.0).

These are the first 10 commits:
- (cb0ad84) [removes preloading from the gem description](https://github.com/fxn/zeitwerk/commit/cb0ad8489ce9fcea32505e5774f5d8849ad522c3)
- (b014363) [documents the Break debugger works with Zeitwerk [skip ci]](https://github.com/fxn/zeitwerk/commit/b014363ddf2d61f9508cea893ddee71e964844fa)
- (a6af0c6) [Add changelog to gemspec](https://github.com/fxn/zeitwerk/commit/a6af0c63409ec42e43fb428cd229a1e3cd8b72a1)
- (a9c6e6d) [Merge pull request #116 from rafbm/patch-1](https://github.com/fxn/zeitwerk/commit/a9c6e6dc91c42b5098ff1459bd189f189d3f6a9a)
- (a44d925) [add some metadata URLs to the gemspec [skip ci]](https://github.com/fxn/zeitwerk/commit/a44d925ab7db7ee9df4229cd943691da483da2fc)
- (5aa874a) [documents reopening third-party namespaces [skip ci]](https://github.com/fxn/zeitwerk/commit/5aa874a528818aa264d602e564a6fc809af14fc5)
- (3793430) [add test coverage for requiring files deep down](https://github.com/fxn/zeitwerk/commit/3793430ce348281a98597ae82277ec2143f1d784)
- (5859f9e) [documents internally why require_relative is not supported](https://github.com/fxn/zeitwerk/commit/5859f9e79dc285b062cc6ce67bd943b3e91b664b)
- (9a189d7) [removes a test covering require_relative](https://github.com/fxn/zeitwerk/commit/9a189d722760cbe24eb618b56d05b7f2215ba7ee)
- (f4eb10b) [more internal docs about require support](https://github.com/fxn/zeitwerk/commit/f4eb10b013b9a39d68a15a1b04103b87271cfbcd)


</details>